langchain/chat_models/base.py

```diff
@@ -82,20 +82,27 @@ def generate(
         )
         try:
             results = [
-                self._generate(m, stop=stop, run_manager=run_manager)
+                self._generate(
+                    m, stop=stop, run_manager=run_managers[0] if run_managers else None
```

Comment:
Now that we'll start to collect events on the run this can be a bit strange, only the first run of the batch will have all events